manuscript/draft.Rmd

```diff
@@ -144,11 +144,14 @@ correcting for observer effects, as described below.
 
 **Richness data.** Bird species richness was obtained from the North American
 Breeding Bird Survey (BBS) [@pardieck2017] using the Data Retriever Python
-package [@morris2013] and rdataretriever R package [@rdataretriever]. The BBS
-data was filtered to exclude all nocturnal, cepuscular, and aquatic species
-[since these species are not well sampled by BBS methods; @hurlbert2005], as
-well as unidentified species, and hybrids. All data from surveys that did not
-meet BBS quality criteria were also excluded.
+package [@morris2013] and rdataretriever R package [@rdataretriever]. BBS
+observations are three minute point counts made at 50 fixed locations along a
+40km route. Here we denote each route as a site and summarize richness as the
```

Comment:
This is likely to be confusing to other readers too.  Should we just switch to "route" throughout?

Reply:
maybe "Each site consists of a 50 point counts along a 40km route" ?